### PR TITLE
Drop audio messages in public chats

### LINKED ISSUE #11977: Audio messages are displayed in public chats
Audio messages should not be displayed in public chats and should be discarded.

### Expected behavior

Audio messages are dropped on receiving when sent in public chats

### Actual behavior

Audio messages are displayed in public chats

### PR DESCRIPTION
**Pull request by qfrank**
Closes [11977](https://github.com/status-im/status-react/issues/11977)

### CODE REVIEW

**Review by cammellos — COMMENTED** [review]
Looks good overall, would you mind adding some tests please?

Thanks!

**Review by cammellos — APPROVED** [review]
Thanks!
It's a negative test, but it's the best we can do I think in this case, well done!

**Comment by cammellos** [comment]
Linting and version conflict, I just noticed

File: protocol/messenger_test.go [hunk 1]
```diff
@@ -309,6 +309,27 @@ func (s *MessengerSuite) TestInit() {
 	}
```
Comment (cammellos) [inline protocol/messenger_test.go:308]: The linter is upset about the formatting, you can fix it by running:
```
goimports -w -local 'github.com/ethereum/go-ethereum,github.com/status-im/status-go,github.com/status-im/markdown' protocol/messenger_test.go
```

That should give you some changes to commit

Comment (qfrank) [reply protocol/messenger_test.go:308]: done @cammellos :)

**Pull request merged**